### ISSUE TITLE
All BNDs have no FORMAT:FT

### ISSUE THREAD

**Issue by tgong1** [DecodeGenetics/graphtyper#127]
Hi,

There are no FT tags in FORMAT fields for ALL BNDs. Do you have any idea why this happen?
Any idea or suggestion is appreciated.

Thank you,
Tingting

**Comment by jingydz** [comment]
Hi, do you have solved  this problem?
I meet the same problem.
![image](https://github.com/DecodeGenetics/graphtyper/assets/45711930/4e5dd661-d07f-4b8d-b62c-7c161000649b)
normal SV type, such as DEL, INS, DUP
![image](https://github.com/DecodeGenetics/graphtyper/assets/45711930/741a85e2-8dc6-47f5-9a2b-f509b77591cf)
but BND, before vcffillter -g FT=PASS
![image](https://github.com/DecodeGenetics/graphtyper/assets/45711930/f0671659-a532-46b6-a427-95f58bfb13dd)
BND, after vcffilter -g FT=PASS
![image](https://github.com/DecodeGenetics/graphtyper/assets/45711930/31483936-2ef6-4d44-b620-71b0a60943ad)

all genotype of BND type changed from 0/0 or 0/1 into "."，that 's true？

**Comment by tgong1** [comment]
Hi,

No, I haven't solve the problem of BND. So I have given up the method of filtering based on FT=PASS. I'm now using the PASS_ratio for filtering.

Best regards,
Tingting

**Comment by jingydz** [comment]
BND：-f "( SVTYPE = BND & QD > 20 & ( ABHet > 0.30 | ABHet < 0 ) & ( AC / NUM_MERGED_SVS ) < 10 & PASS_AC > 0 & PASS_ratio > 0.1 )" 

-f, --info-filter     specifies a filter to apply to the info fields of records, removes alleles which do not pass the filter

-g, --genotype-filter specifies a filter to apply to the genotype fields of records

If you only use the PASS_ratio, then it only filter the BND records but not sample genotype?  can it filter unpassed genotype?

**Comment by tgong1** [comment]
@jingydz Yes, the filtering I'm using now is for SV sites only. While I have also tried on filtering BND based on GQ>40 (you have to set a threshold here), and other types based on FT=PASS. Hope that helps.

**Comment by jingydz** [comment]
Thank you again. Finally I used "( GQ > 20 ) & ( AD > 10 )" for BND and "FT=PASS" for other types.
